IoTMining/utils.py

```diff
@@ -95,6 +95,27 @@
             "dHall_Light",
             "Kitchen_Light")
 
+labelTestSet = ("Kitchen_Light",
+                "R1room",
+                "R2room",
+                "Liv_Light",
+                "dHall_Light")
+
+basketsKeySet = ("00", "01", "02", 
```

Comment:
I think this should be done programmatically. We have 0-6 for days of a week, and count for the timePartitionMap size to build this. Later on, if we change the timePartitionMap, we don't have to redo this